src/showingpreviously/cinemas/savoy.py

```diff
@@ -0,0 +1,130 @@
+import json
+import re
+from datetime import datetime, timedelta
+from bs4 import BeautifulSoup
+
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import UK_TIMEZONE, UNKNOWN_FILM_YEAR, STANDARD_DAYS_AHEAD
+
+
+CINEMAS_URL = 'http://www.savoycinemas.co.uk/'
+CHAIN = Chain('Savoy Cinemas')
+EVENTS_INLINE_PATTERN = re.compile(r'var\s+eventsInline\s*=\s*(?P<events_inline>(?:.|\n|\r)*?]);')
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+def parse_film_name(name: str) -> (str, [str]):
+    attributes = {'format': []}
+    # ending
+    for removal in [' U', '(PG)', '(12A)', '(12)', '(15)', '(18)', '2D', '3D', '4K', '(WITH SANTA)']:
```

Comment:
There is also:
- ` WITH SANTA CLAUS` (no brackets)
- `WITH SANTA` (no brackets)
- `ROH 22-23 SEASON` (maybe make this generic to `ROH YY-YY+1 SEASON`?)
- `NTL:` (this one is a *starting* one)
- `ROH 22-23 SEASON:` (starting)
- `ROH 2022/23 SEASON: ` (starting)
- `ROH 22-23 SEASONS:` (starting)
- `W/SANTA`
- `NATIONAL THEATRE LIVE :` (starting)